README.md

```diff
@@ -515,14 +515,13 @@ Sets or clears a RabbitMQ [policy](https://www.rabbitmq.com/parameters.html#poli
 
 - `:set` sets a `policy`
 - `:clear` clears a `policy`
-- `:list` lists `policy`s
 
 ### Examples
 
 ``` ruby
 rabbitmq_policy "queue-length-limit" do
   pattern "^limited\\.*"
-  parameters ({"max-length" => "3000"})
```

Comment:
We will have to bump the minor over this. Can `parameters` be aliased for backwards compatibility? (it's OK if it's not worth doing in your opinion)

Reply:
Ah, I see a deprecated alias defined in a different file. Sorry about the noise.

Reply:
I was thinking we may even need to bump the major, in case anyone out there is using the `:list` action.

Reply:
We can ship that soon with a default RabbitMQ version revision. 3.7 and 3.8 are [both out
of support](https://www.rabbitmq.com/versions.html) by now.